client/my-sites/upgrade-nudge/index.jsx

```diff
@@ -139,3 +137,13 @@ export default React.createClass( {
 		);
 	}
 } );
+
+function mapStateToProps( state ) {
+	const site = getSelectedSite( state );
+
+	return {
+		site,
+	};
+}
+
+export default connect( mapStateToProps )( localize( UpgradeNudge ) );
```

Comment:
No strong objection - I just tend to break it out in the first place to avoid much change as things are added to `mapStateToProps` or `mapDispatchToProps`. 
The inlined pattern is growing on me actually :P